receiver/prometheusexecreceiver/factory.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheusexecreceiver
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/config/configerror"
+	"go.opentelemetry.io/collector/config/configmodels"
+	"go.opentelemetry.io/collector/consumer"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusexecreceiver/subprocessmanager"
+)
+
+// Factory for prometheusexec
+const (
+	// Key to invoke this receiver (prometheus_exec)
+	typeStr = "prometheus_exec"
+
+	defaultMetricsPath        = "/metrics"
+	defaultCollectionInterval = 60 * time.Second
+	defaultScrapeTimeout      = 10 * time.Second
```

Comment:
Declare these constants (defaultMetricsPath, defaultScrapeTimeout) in the file where they are used